config/keyqueue.js

```diff
@@ -299,6 +380,13 @@ exports.keyqueueReacted = async function(msg, emoji, member) {
                             color: 0x00ab30
                         }
                     });
+
+                    if (CONFIG.SystemConfig.servers[msg.guildID].keyqueue.keyping) {
+                        let pingChannel = await CONSTANTS.bot.getChannel(CONFIG.SystemConfig.servers[msg.guildID].keyqueue.pingchannel);
+                        let pingRole = CONFIG.SystemConfig.servers[msg.guildID].keyqueue.pingrole;
+
+                        await pingChannel.createMessage(`<@&${pingRole}> ${member.mention} has ${numItems} <:${emojiText}> to pop!`);
+                    }
                 })
                 
                 setTimeout(() => {
```

Comment:
First, great first PR. Code is clean and well managed, compared to some of the garbage I've written.

 A few additional things that need to happen before this gets merged into main:

1. Look at how the `guildRoleDelete` event is managed. To make sure that no 'garbage data' is left in the bot when the `pingrole` is added, you should remove it from the config from that server when that role is deleted. See other examples in the codebase.
2. Look at how the `channelDelete` event is managed. Same comment as above.
3. We need to display `keyqueue` information in the *settings* page of the bot. I.E. when the user types `.showconfig` it should have a section attributed to `keyqueue`. This should then be navigable to with `.showconfig keyqueue`, where the properties for `keyqueue` are shown. There are plenty of examples for this in the relevant file.
4. Do a pass through and make *100% sure* that no errors can POSSIBLY go `uncaught` and break the bot. This code will be run on over 60 servers with `>30,000` people affected. If you fail to catch an error and we don't prevent it now, it can kill some events later.
5. Once all this is done, send me a list of updated fields to the database. I will run the update, then pull your code and run it locally and test it. If all works, I'll merge and ship -- otherwise, I'll leave some more comments (and DM you) to lyk what broke before we merge.


Looks wonderful man! Thank you for your contribution!

Reply:
Hi,

1. Fixed in 4cdf940da4405ca045dbcf25cd074cbe533a5766 and 4c711729234929171355aadb39c7a2bd5b8e58cb
2. Fixed in 4cdf940da4405ca045dbcf25cd074cbe533a5766
3. Fixed in be63180d0e7817f2e59a208b987bf56bdf228f46
4. I have done my best to test everything on my end and it all seems to be working completely fine.
5. I commented on your original comment regarding the database fields however I will also reach out to you on discord regarding this.

Thanks 😄